pkg/querier/queryrange/queryrange.proto

```diff
@@ -59,3 +60,7 @@ message Extent  {
   string trace_id = 4 [(gogoproto.jsontag) = "-"];
   google.protobuf.Any response = 5 [(gogoproto.jsontag) = "response"];
 }
+
+message CacheDirectives {
+  bool disabled = 1;
```

Comment:
Sorry for being annoying, but please let's not use "negative" case for the name, unless there is a good reason to (eg. need for default value in Go). I would suggest to be little more descriptive and call it `cachingEnabled`.

(For discussion, not a suggestion: Since this message is called `CachingDirectives`, should flag inside be named after "no-store" directive? Personally I wouldn't go this far, but I would also just call it `CachingOptions`.)

Reply:
Hi @pstibrany. It is better to use negative here since we need a default value `false`, https://github.com/cortexproject/cortex/pull/3054#discussion_r479318496.

`CachingOptions` seems a better name! Thanks, let me update the pr.

Reply:
> Hi @pstibrany. It is better to use negative here since we need a default value `false`, [#3054 (comment)](https://github.com/cortexproject/cortex/pull/3054#discussion_r479318496).

Sorry, my mistake. I remember having exactly this discussion with Marco, but forgot it was for this PR :)